Casks/asana.rb

```diff
@@ -0,0 +1,26 @@
+cask "asana" do
+  version "1.1.0"
+  sha256 :no_check
+
+  url "https://desktop-downloads.asana.com/darwin_x64/prod/latest/Asana.dmg"
+  name "Asana"
+  desc "Easiest way to manage team projects and tasks"
```

Comment:
Remove marketing terms from [`desc`](https://docs.brew.sh/Cask-Cookbook#stanza-desc):
```suggestion
  desc "Manage team projects and tasks"
```